### PR TITLE
Cleanup msgs generation

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
+protobuf_generate_cpp(msgs_SRCS msgs_HDR msgs.proto)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 rock_library(gazebo_thruster
-    SOURCES GazeboThruster.cpp
-    HEADERS GazeboThruster.hpp
-#    DEPS thruster_types_msgs
-    DEPS_PKGCONFIG gazebo protobuf)
+    SOURCES GazeboThruster.cpp ${msgs_SRCS}
+    HEADERS ${msgs_HDR}
```
Comment (Thomio-Watanabe) [inline src/CMakeLists.txt]: ${msgs_HDR} copy the build/ folder into include/gazebo_thruster.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -1,23 +1,14 @@
...
+    DEPS_PLAIN Boost
+    DEPS_PKGCONFIG gazebo sdformat protobuf)
+
+install(FILES msgs.proto
```
Comment (Thomio-Watanabe) [inline src/CMakeLists.txt]: We actually install msgs.pb.h

**Comment by Thomio-Watanabe** [comment]
Here. Rename protobuf file and install it

**Comment by doudou** [comment]
Not agreeing on that one. What you did here is install a file

```
gazebo_thruster/thrusters.pb.h
```

for types called gazebo_thruster.msgs.Thruster and gazebo_thruster.msgs.Thrusters

The convention in gazebo is path/to/file for path.to.file (which would mean having two proto files installed in gazebo_thrusters/msg/thruster.pb.h and gazebo_thrusters/msg/thrusters.pb.h). In addition, the proto file goes into the proto/ subdirectory of msgs/

So either we split the file and match the convention, or keep a single file and call it msgs.proto (since it contains everything from gazebo_thrusters.msgs).

**Comment by Thomio-Watanabe** [comment]
Ok, msgs.pb.h it is. But where we install msgs.proto ?

**Comment by Thomio-Watanabe** [comment]
Amend last commit

**Comment by doudou** [comment]
The proto files go in include/PROJECT/proto... Just amended the last commit for that.

**Comment by Thomio-Watanabe** [comment]
Good for me. Merged

**Pull request merged**